application/src/main/resources/application.yml

```diff
@@ -10,7 +10,6 @@ hono:
     singleTenant: false
     insecurePortEnabled: false
     bindAddressInsecurePort: 127.0.0.1
```

Comment:
why do we configure these two variables (`insecurePortEnabled` and `bindAddressInsecurePort``)?